contracts/vaults/RibbonThetaVault.sol

```diff
@@ -608,6 +649,46 @@ contract RibbonThetaVault is OptionsVaultStorage {
         }
     }
 
+    /*
+     * @notice Helper function that transfers management fees and performance fees from previous round.
+     * @param currentLockedBalance is the balance we are about to lock for next round
+     * @return vaultFee is the fee deducted
+     */
+    function _collectVaultFees(uint256 currentLockedBalance)
+        private
+        returns (uint256 vaultFee)
+    {
+        // Take management fee
+        if (managementFee > 0) {
+            uint256 managementFeeInAsset =
+                currentLockedBalance.mul(managementFee).div(100 * 10**6);
+            vaultFee = managementFeeInAsset;
+        }
```

Comment:
We should also take management fee only when the vault is profitable for the week (just like the performance fee)